### PR TITLE
Trial error

### PR DESCRIPTION
**Pull request by mrcallenmatias**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -106,42 +119,52 @@ export default class App extends React.Component {
...
                 <Show profile={profile} deleteProfile={this.deleteProfile} />
               );
             }}
           />
-          <Route exact path="/create-profile" component={Register} />
-          <Route exact path="/posts/view" component={PostList} />
-          <Route exact path="/posts/new" component={NewPost} />
+
+          {/* REGISTER */}
+          <Route
```
Comment (iameddieyayaya) [inline app/javascript/components/App.js:161]: formatting is off

**Review by iameddieyayaya — CHANGES_REQUESTED** [review]
Looks petty good

File: app/controllers/registrations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class RegistrationsController < Devise::RegistrationsController
+  protected
+
+  def after_sign_up_path_for(resource)
+    '/create-profile' # Or :prefix_to_your_route
+  end
+end
```
[the remark this answers is not in the document]
Comment (mrcallenmatias) [reply app/controllers/registrations_controller.rb]: solve

File: app/javascript/components/App.js (hunk 1 above)
Comment (mrcallenmatias) [reply app/javascript/components/App.js:161]: solve

**Review by iameddieyayaya — APPROVED** [review]

**Pull request merged**